.changeset/modern-experts-dream.md

```diff
@@ -0,0 +1,7 @@
+---
+"@astrolicious/studiocms": patch
+---
+
+- StudioCMS-Dashboard:
+
+Update UnoCSS to `0.61.5` and DaisyUI Preset to `0.1.1`
```

Comment:
Just a tiny suggestion, aside from this LGTM 🫡
```suggestion
Updates UnoCSS to `0.61.5`, DaisyUI Preset to `0.1.1` and adds changeset workflow.
```

Reply:
Changeset is not part of the package itself, but part of the repo... should it be included within the packages changelog... :thinking:

Reply:
hmmm... you're right, also which is going to be the way we're going write changests infinitive or present tense? 🤔

Reply:
Well, its a clean slate.... so honestly I'm not sure really what would be best overall.

![image](https://github.com/user-attachments/assets/1201255b-8b52-42ca-b18f-778c39e541e4)

Reply:
Let's keep them as you started.